index.bs

```diff
@@ -1168,6 +1168,33 @@ The authenticator MUST return both the <a>authenticatorData</a> and the raw sign
 MUST return <a>clientDataJSON</a>, <a>authenticatorData</a> and the signature to the RP. The first two are returned in the
 `clientData` and `authenticatorData` members respectively of the {{WebAuthnAssertion}} and {{WebAuthnAttestation}} structures.
 
+### Verifying a signature ### {#authenticator-signature-verification}
+
+This section specifies the algorithm for verifying a signature assertion.
+
+Upon receiving a signature assertion in the form of a {{WebAuthnAssertion}} structure, the [RP] shall:
+
+1. Perform JSON decoding to extract the {{ClientData}} used for the assertion from the {{WebAuthnAssertion/clientData}}.
```

Comment:
hm, if WebAuthnAssertion.clientData contains JSON-serialized data, perhaps we should rename it serializedClientData